### PR TITLE
feat: product page

### PR DESCRIPTION
**Pull request by bhtibrewal**
Added product page
Pages to check
- [ ] src/components/card/TextOverMediaCard.jsx
- [ ] src/pages/product_page/ProductPage.jsx

### CODE REVIEW

File: src/components/card/BasicCard.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          <div class="price-sec">
+            <h3>$18</h3>
+            <span class="strike-price">$22</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="card-actions">
+        <button class="btn btn-primary">
+          <i class="fa-solid fa-cart-shopping"></i>
```
Comment (Naman-Saxena1) [inline src/components/card/BasicCard.jsx:30]: Everywhere, it should be className instead of class.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
In Navbar, you can position wishlist and cart badges on top-right with circle background. Everything else looks fine.

File: src/components/card/BasicCard.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <div class="card w-30 basic">
+      <button class="icon favourite-icon">
+        <i class="fa-regular fa-heart fa-2x"></i>
+      </button>
+      <div class="content">
+        <img
+          class="card-img"
+          src="https://render.fineartamerica.com/images/images-new-artwork/images-medium-5/la-porta-rossa-sulla-salita-guido-borelli.jpg?v=2"
+          alt=""
```
Comment (gauravsinhaweb) [inline src/components/card/BasicCard.jsx:11]: add text to img, apply this for add

File: src/pages/product_page/ProductPage.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            <div>Abstraction</div>
+          </div>
+        </div>
+
+        {/*  */}
+        <div className="price-sec">
+          <h3>Price</h3>
+          <div>
+            <input value="" type="range" min="0" max="1000" step="50" />
+            <span>{}</span>
```
Comment (gauravsinhaweb) [inline src/pages/product_page/ProductPage.jsx:48]: line 48 is not required ig

File: src/pages/product_page/ProductPage.jsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        <div className="grid-3 products-grid">
+          {Array(12)
+            .fill("")
+            .map(() => {
+              return (
+                <TextOverMediaCard
+                  item={{
+                    item_name: "La Porta Rossa Sulla Salita Art Print",
+                    item_by: "Guido Borelli",
+                    item_desc: "",
```
Comment (gauravsinhaweb) [inline src/pages/product_page/ProductPage.jsx:80]: item desc is empty.

**Review by gauravsinhaweb — APPROVED** [review]
good to merge after changes!

File: src/pages/product_page/ProductPage.jsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+          <div>
+            <input value="" type="range" min="0" max="1000" step="50" />
+            <span>{}</span>
+          </div>
+        </div>
+
+        {/*  sort by dropdown  */}
+        <div className="dropdown-container sortBy-con">
+          <button className="btn dropdown-btn" onClick="">
+            Sort By
```
Comment (Ankur9669) [inline src/pages/product_page/ProductPage.jsx:55]: onClick is empty

File: src/pages/product_page/ProductPage.jsx (hunk 3 above)
Comment (bhtibrewal) [reply src/pages/product_page/ProductPage.jsx:48]: I will add a variable there

**Review by Ankur9669 — APPROVED** [review]
Code looks good as always , it is good to merge

File: src/components/card/BasicCard.jsx (hunk 1 above)
Comment (bhtibrewal) [reply src/components/card/BasicCard.jsx:30]: Thank you, yes i will change it

**Pull request merged**